src/base/lane.h

```diff
@@ -59,6 +59,9 @@ class Lane : public maliput::geometry_base::Lane {
   const geometry::LaneGeometry* lane_geometry() const { return lane_geometry_.get(); }
 
  private:
+  static constexpr bool kUseLaneBoundaries{true};
+  static constexpr bool kUseSegmentBoundaries{false};
```

Comment:
```suggestion
  static constexpr bool kUseLaneBoundaries{true};
  static constexpr bool kUseSegmentBoundaries{!kUseLaneBoundaries};
```

Reply:
Does sit make sense to promote this to lane_data.h?

Reply:
It won't be used by the user at a maliput api level. Maybe could be added to geometry_base::lane.

Reply:
Sounds good to me.